docs/tutorial/system.rst

```diff
@@ -0,0 +1,72 @@
+============================
+System: Pendulum on Cart
+============================
+
+A pendulum is fixed on a cart, which can move in the horizontal direction.
+
+The cart has a mass :math:`M`. 
+The friction between the cart and the surface causes
+a frictional force :math:`F_R = D \cdot \dot{s}`,
+in opposite direction as the velocity :math:`\dot{s}` of the cart.
+
+
+The pendulum has a mass :math:`m`, a moment of intertia :math:`J`,
+a length :math:`l` and an angle of deflection :math:`\varphi`.
+The friction in the joint where the pendulum is mounted on the cart 
+causes a frictional torque :math:`M_R = d \cdot \dot{\varphi}`,
+in opposite direction as the speed of rotation :math:`\dot{\varphi}`.
+
+The task is to control the position :math:`s` of the cart 
+and the deflection angle :math:`\varphi` of the pendulum.
```

Comment:
So that the pendulum is stabilized in either its up- or downside configuration.

Reply:
That is not entirely correct, since stabilizing the upward position is ignored during this tutorial.